src/main/kotlin/no/nav/arrangor/ansatt/AnsattController.kt

```diff
@@ -61,6 +61,14 @@ class AnsattController(
 		}
 	}
 
+	@ProtectedWithClaims(issuer = Issuer.AZURE_AD)
+	@PostMapping("tilganger")
+	fun fjernTilgangerHosArrangor(
```

Comment:
Det finnes en egen controller, `AnsattControllerServiceUser`, der det er samlet endepunkter som skal brukes med servicebrukercontext. Kanskje flytte denne dit?

Reply:
Ja det høres riktig ut. Jeg tolket ikke den som en kontroller it i fra navnet 😅